Wordle.Api/Wordle.Api/Controllers/PlayerController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class PlayerController : ControllerBase
+    {
+        private readonly PlayerService _playerService;
+
+        public PlayerController(PlayerService playerService)
+        {
+            _playerService = playerService;
+        }
+
+
+        [HttpGet("GetTopTenPlayers")]
+        public async Task<IEnumerable<Player>> GetTopTenPlayers()
+        {
+            return await _playerService.GetTopTen();
+        }
+
+        [HttpPost("AddPlayerFromBody")]
```

Comment:
```suggestion
        [HttpPost("UpsertPlayer")]
```
You may just be updating a player here too, so 'upsert' or 'update' might be a better endpoint name. Or just 'player'

Reply:
Thanks for your feedback Meg!